docker-images/artifacts/kafka-thirdparty-libs/4.0.x/pom.xml

```diff
@@ -17,6 +17,8 @@
 
     <properties>
         <strimzi-oauth.version>0.15.0</strimzi-oauth.version>
+        <strimzi-metrics-reporter.version>0.2.0</strimzi-metrics-reporter.version>
+        <prometheus.version>1.3.1</prometheus.version>
```

Comment:
You updated `strimzi-metrics-reporter` to 0.2.0, so you also need to update `prometheus` to 1.3.6.